Sources/Fuzzilli/Core/CodeGenerators.swift

```diff
@@ -255,6 +255,27 @@ public let CodeGenerators: [CodeGenerator] = [
         b.callMethod(methodName!, on: obj, withArgs: arguments)
     },
 
+    CodeGenerator("MethodCallWithSpreadGenerator", input: .object()) { b, obj in
+        guard b.mode != .conservative else { return }
```

Comment:
Please add a comment here and in the other functions like `We cannot currently track element types of Arrays and other Iterable objects and so cannot properly determine argument types when spreading. For that reason, we don't run this CodeGenerator in conservative mode`